### PR TITLE
update to 3.7.1

### PR DESCRIPTION
**Pull request by ocefpaf**
Alternative to #221

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,10 +24,6 @@ source:
       - patches/0013-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch
       - patches/0014-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
       - patches/0015-Disable-new-dtags-in-unixccompiler.py.patch
-      # AppVeyor does not have the 10.0.16299 SDK installed, leave as is
-      # This is actually something else now
-      #- patches/0016-Switch-from-WinSDK-10.0.17134.0-to-10.0.16299.0.patch
-      - winsdk.patch  # [win]
```
Comment (jjhelmus) [inline recipe/meta.yaml]: I'm fairly certain that this will be needed, but lets let AppVeyor run and see for sure.

Comment (ocefpaf) [reply recipe/meta.yaml]: I'll restore it if we need but, I may be confused b/c I did multiple tests early this morning, I think it passed on my AppVeyor account without it.

Comment (ocefpaf) [reply recipe/meta.yaml]: And of course you are right :grimacing: I looked at the wrong log:

https://ci.appveyor.com/project/ocefpaf/python-feedstock/builds/20266954

File: recipe/meta.yaml [hunk 2]
```diff
@@ -50,13 +50,12 @@ source:
...
-    sha256: 65facaa26aad903889f1ef8e4d14542a0c0d142940a8cf47d6f57516fba6a41f         # [win]
-  - url: https://github.com/python/cpython-bin-deps/archive/openssl-bin-1.1.0h.zip   # [win]
-    folder: externals/openssl-bin-1.1.0h                                             # [win]
-    sha256: 8388d90b06b878a7fd980a46264d2d2aca2900ce36dd6834cf85861921534c35         # [win]
+  - url: https://github.com/python/cpython-source-deps/archive/openssl-1.1.0i.zip    # [win]
+    folder: externals/openssl-1.1.0i                                                 # [win]
+    sha256: 2280b9b4cba4dfcffc96d41860c79bdfd2d35945f9dd5e5a545b467ef6f96603         # [win]
+  - url: https://github.com/python/cpython-bin-deps/archive/openssl-bin-1.1.0i.zip   # [win]
+    folder: externals/openssl-bin-1.1.0i                                             # [win]
+    sha256: 71db5e71d2d6307379bac6c581492d37b65f2ad67fea207f6c0222150a4a46a7         # [win]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @jjhelmus and @mariusvniekerk are we (conda-forge) good for this change? Sorry for the dummy question but I've been out of the loop in the python package for a while.

Comment (jjhelmus) [reply recipe/meta.yaml]: This is a good change.  OpenSSL was updated to 1.1.0i for Windows build in the 3.7.1 release, see [bpo-34405](https://bugs.python.org/issue34405)

**Comment by jjhelmus** [comment]
LGTM

**Comment by jjhelmus** [comment]
Thanks for updating this @ocefpaf!

**Comment by ocefpaf** [comment]
Thanks for the merge. I'll work on the 3.6 update tomorrow (or maybe later today).

**Pull request merged**